### PR TITLE
Removing some deprecated components flagged when turning tslint deprecation flag to true in oufr package

### PR DESCRIPTION
**Pull request by khmakoto**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #6310 
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Removing some deprecated components flagged when turning tslint deprecation flag to true in oufr package.

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx [hunk 1]
```diff
@@ -51,9 +51,9 @@ export class DropdownBase extends BaseComponent<IDropdownInternalProps, IDropdow
     options: [] as any[]
   };
 
-  private _host = createRef<HTMLDivElement>();
+  private _host = React.createRef<HTMLDivElement>();
   private _focusZone = createRef<IFocusZone>();
```
Comment (kenotron) [inline packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx]: Why not convert this createRef as well?

Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx]: Right; remove the import and you'll have to fix em :)

If there's a good reason to keep, we need to discuss why (and add lots of comments) but I think this is just an oversight.

Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx]: maybe i can just do it here.

Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx]: fixed.

**Pull request merged**